frontend/apps/marketing/src/components/link/link.module.scss

```
@@ -0,0 +1,53 @@
+a.link {
+  margin-top: 0;
+}
+
+// Currently purple color styles are match component-library styles completely so we are not editing it.
+//a.link-purple {
+//}
+
+a.link-white {
+  color: var(--text-neutral-white-fixed);
+
+  &:hover,
+  :active,
+  :visited {
+    color: var(--neutral-gray-10);
+  }
+
+  &:focus-visible {
+    color: var(--text-neutral-white-fixed);
+  }
+}
+
+a.link-l {
```

Nit: can you add the comment structure here like we have in the other components? (And `// Link margin` on line 39)
```suggestion
// Link sizes
a.link-l {
```